deploy/frontend-stage-internal-test-job.yaml

```diff
@@ -0,0 +1,131 @@
+          restartPolicy: Never
+          volumes:
+            - name: sel-shm
+              emptyDir:
+                medium: Memory
+          containers:
+            - name: curiosity-frontend-stage-post-deploy-tests-${IMAGE_TAG}-${UID}
+              args:
+                - run
+              env:
+                - name: ENV_FOR_DYNACONF
+                  value: ${ENV_FOR_DYNACONF}
+                - name: IQE_PLUGINS
+                  value: ${IQE_PLUGINS}
+                - name: IQE_MARKER_EXPRESSION
+                  value: ${IQE_MARKER_EXPRESSION}
+                - name: IQE_FILTER_EXPRESSION
+                  value: ${IQE_FILTER_EXPRESSION}
+                - name: IQE_LOG_LEVEL
+                  value: ${IQE_LOG_LEVEL}
+                - name: IQE_REQUIREMENTS
+                  value: ${IQE_REQUIREMENTS}
+                - name: IQE_REQUIREMENTS_PRIORITY
+                  value: ${IQE_REQUIREMENTS_PRIORITY}
+                - name: IQE_TEST_IMPORTANCE
+                  value: ${IQE_TEST_IMPORTANCE}
+                - name: IQE_PARALLEL_WORKER_COUNT
+                  value: ${IQE_PARALLEL_WORKER_COUNT}
+                - name: IQE_RP_ARGS
+                  value: "true"
+                - name: IQE_IBUTSU_SOURCE
+                  value: "rhsm-stage-${IMAGE_TAG}"
+                - name: DYNACONF_IQE_VAULT_LOADER_ENABLED
+                  value: "true"
+                - name: DYNACONF_IQE_VAULT_VERIFY
+                  value: "true"
+                - name: DYNACONF_IQE_VAULT_URL
+                  valueFrom:
+                    secretKeyRef:
+                      key: url
+                      name: iqe-vault
+                      optional: true
+                - name: DYNACONF_IQE_VAULT_MOUNT_POINT
+                  valueFrom:
+                    secretKeyRef:
+                      key: mountPoint
+                      name: iqe-vault
+                      optional: true
+                - name: DYNACONF_IQE_VAULT_ROLE_ID
+                  valueFrom:
+                    secretKeyRef:
+                      key: roleId
+                      name: iqe-vault
+                      optional: true
+                - name: DYNACONF_IQE_VAULT_SECRET_ID
+                  valueFrom:
+                    secretKeyRef:
+                      key: secretId
+                      name: iqe-vault
+                      optional: true
+                - name: DYNACONF_main__use_beta
+                  value: "yes"
+                - name: DYNACONF_main__use_browser
+                  value: "chrome"
+                - name: DYNACONF_browsers__chrome__webdriver
+                  value: "Remote"
+                - name: DYNACONF_browsers__chrome__command_executor
+                  value: "http://10.0.148.118:80/wd/hub"
+                - name: DYNACONF_browsers__chrome__webdriver_options__desired_capabilities__chromeOptions__args
+                  value: "['--headless','--window-size=1920,1080']"
```

Comment:
There is a problem with this. The Selenium Grid has disappeared from our OpenStack. Somehow it got deleted.